wsmaster/che-core-api-logger/src/main/java/org/eclipse/che/api/logger/LoggerService.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.logger;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static org.eclipse.che.dto.server.DtoFactory.newDto;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import org.eclipse.che.api.core.NotFoundException;
+import org.eclipse.che.api.core.rest.Service;
+import org.eclipse.che.api.logger.shared.dto.LoggerDto;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Defines Logger REST API. It allows to manage the loggers (with log level) dynamically.
+ *
+ * @author Florent Benoit
+ */
+@Api(value = "/logger", description = "Logger REST API")
+@Path("/logger")
```

Comment:
can you make sure this service is protected with authentification in multi-user mode?

Reply:
for now it was not enabled in multi user mode

Reply:
What is your plan for that?

Reply:
permission mechanism added